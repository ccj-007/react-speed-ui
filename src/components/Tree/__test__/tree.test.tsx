import { render } from '@testing-library/react';
import '@testing-library/jest-dom';
import { Tree } from '../../index';

describe('test Tree component', () => {
  it('should render', () => {
    const { asFragment } = render(<Tree />);
    expect(asFragment()).toMatchSnapshot();
  });
});
