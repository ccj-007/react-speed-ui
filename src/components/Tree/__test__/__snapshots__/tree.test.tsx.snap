// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Tree component should render 1`] = `
<DocumentFragment>
  <div
    class="speed-tree"
  />
</DocumentFragment>
`;
